guides/source/upgrading_ruby_on_rails.md

```diff
@@ -70,6 +70,66 @@ Upgrading from Rails 5.0 to Rails 5.1
 
 For more information on changes made to Rails 5.1 please see the [release notes](5_1_release_notes.html).
 
+### Moving from Erubis to Erubi
+
+If your application uses the `ActionView::Template::Handlers::Erubis` class,
+you should move your code to use `ActionView::Template::Handlers::Erubi` instead.
+
+Erubis is deprecated and will be removed from Rails 5.2.
+
+See [#27757](https://github.com/rails/rails/pull/27757) for more details.
+
+### Passing string to `:if` and `:unless` options when setting or skipping callbacks
+
+When setting or skipping callbacks, you can specify conditions with the `:if`
+and `:unless` options.
+
+If your application provides these options as strings, you should move your code
+to use symbols, arrays of symbols or Procs instead. For instance code like:
+
+```ruby
+class Post < ApplicationRecord
+  before_save :set_default_title, unless: 'title.present?'
+  skip_callback :save, :before, :set_default_title, if: 'persisted?'
+
+private
+
+  def set_default_title
+    self.title = 'No title'
+  end
+end
+```
+
+is deprecated in Rails 5.1. Instead, you can replace it with code like:
+
+```ruby
+class Post < ApplicationRecord
+  before_save :set_default_title, unless: -> { title.present? }
+  skip_callback :save, :before, :set_default_title, if: :persisted?
+
+private
+
+  def set_default_title
+    self.title = 'No title'
+  end
+end
+```
+
+See [commit 0952552](https://github.com/rails/rails/commit/0952552) for more details.
+
+### Using `:controller` and `:action` in the routes
+
+The special "wildcard routes" symbols `:controller` and `:action` are
+deprecated in Rails 5.1.
```

Comment:
We don't and I don't think we need to. The deprecations should be clear about what they should do. We only address breaking changes in the upgrade guide or things they need to do or configure that are not being told by deprecations, like generating the new framework default files, or in the case of rails 5.1, running `rails db:schema:dump` after the version is changed and committing that file.

I'd not include all of those three entries here since deprecation does their job.